src/services/import/enex.js

```diff
@@ -221,6 +221,17 @@ function importEnex(taskContext, file, parentNote) {
 
         content = extractContent(content);
 
+        // This boilerplate is added by the Evernote import of OneNote notes (and preserved in ENEX export)
+        const replaced = content.replace(/<div>\s*<p>\s*<\/p>\s*<p>\s*Created with OneNote.\s*<\/p>\s*<\/div>\s*$/m, "");
+        if (replaced != content) {
```

Comment:
I think this could simply check `content` to see if it contains the string, then if it does, then replace it and add the attribute. No need to clone the entire note then compare them.